.doc_gen/templates/zonbook/example_language_template.xml

```diff
@@ -44,7 +44,13 @@
             {{- end}}
             {{- range $version.Excerpts}}
             <block>
-                <para>{{.Description}}</para>
+                {{- if .Description}}
+                    {{- if and (lt 15 (len .Description)) (eq (slice .Description 0 15) "<programlisting")}}
```

Comment:
`{{- if .Description | strings.HasPrefix "<programlisting" }}`

Reply:
Heh, I forgot I'd added that custom function. Much better!

Reply:
I think that's just a builtin? `strings` aiui is always exported in text/template? Or maybe it's just such an obvious convention, golang ought to do that...